core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackToStackV4ResponseConverter.java

```diff
@@ -89,7 +91,7 @@ public StackV4Response convert(Stack source) {
         response.setStatus(source.getStatus());
         response.setTerminated(source.getTerminated());
         response.setStatusReason(source.getStatusReason());
-        response.setInstanceGroups(converterUtil.convertAll(source.getInstanceGroups(), InstanceGroupV4Response.class));
```

Comment:
maybe its better to keep this conversion and call `collectInformationsFromActualHostgroup` after this line